File: mihabitat-web/src/main/java/com/bstmexico/mihabitat/web/dto/reportes/SaldoDetalle.java
package com.bstmexico.mihabitat.web.dto.reportes;

import java.math.BigDecimal;
import java.util.Date;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonFormat.Shape;

/**
 * @author Pablo Cruz Santos
 * @version 1.0
 * @since 2015
 */
public class SaldoDetalle {

	private Long id;

	@JsonFormat(shape = Shape.STRING, pattern = "dd-MM-yyyy", timezone = "America/Mexico_City")
	private Date fecha;

	private BigDecimal monto;
	
	private String descripcion;

	public SaldoDetalle() {
		super();
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Date getFecha() {
		return fecha;
	}

	public void setFecha(Date fecha) {
		this.fecha = fecha;
	}

	public BigDecimal getMonto() {
		return monto;
	}

	public void setMonto(BigDecimal monto) {
		this.monto = monto;
	}

	public String getDescripcion() {
		return descripcion;
	}

	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}
}

File: mihabitat-web/src/main/webapp/recursos/js/app/reportes/saldo-detalle.js
var SaldoDetalle = function() {
    
    var self = this;
    
    self.id = ko.observable();
    self.fecha = ko.observable();
    self.monto = ko.observable();
    self.descripcion = ko.observable();
    
    self.limpiar = function() {
        self.id(undefined);
        self.fecha(undefined);
        self.monto(undefined);
        self.descripcion(undefined);
    }
    
    self.cargar = function(data) {
        self.id(data ? data.id : undefined);
        self.fecha(data ? moment(data.fecha, 'DD-MM-YYYY').format('YYYY-MM-DD') : undefined);
        self.monto(data ? data.monto : undefined);
        self.descripcion(data ? data.descripcion : undefined);
    }
}